buildspec.json

```diff
@@ -1,88 +1,67 @@
+                "windows-x64": "c13a14a1acc4224b21304d97b63da4121de1ed6981297e50496fbc474abc0503"
             }
         },
         "qt6": {
-            "version": "2022-08-02",
+            "version": "2023-04-12",
             "baseUrl": "https://github.com/obsproject/obs-deps/releases/download",
-            "label": "Pre-built Qt6",
+            "label": "Pre-Built Qt6",
             "hashes": {
-                "macos-x86_64": "a83f72a11023b03b6cb2dc365f0a66ad9df31163bbb4fe2df32d601856a9fad3",
-                "macos-arm64": "2f30af90c049670a5660656adbb440668aa1b0567f75a5f29e1def9108928403",
-                "macos-universal": "252e6684f43ab9c6f262c73af739e2296ce391b998da2c4ee04c254aaa07db18",
-                "windows-x64": "e5509b54196a3f935250cc4b9c54160c8e588fd0f92bc078a2a64f9d9e2e4e93",
-                "windows-x86": "24fc03bef153a0e027c1479e42eb08097a4ea1d70a4710825be0783d0626cb0d"
+                "macos": "eb7614544ab4f3d2c6052c797635602280ca5b028a6b987523d8484222ce45d1",
+                "windows-x64": "4d39364b8a8dee5aa24fcebd8440d5c22bb4551c6b440ffeacce7d61f2ed1add"
             },
-            "pdb-hashes": {
-                "windows-x64": "60e5b1d2bc4d7c431bc05f14e3b1e85e088788c372fa85f58717cd6c49555a46",
-                "windows-x86": "f34d1a89fc85d92913bd6c7f75ec5c28471d74db708c98161100bc8b75f8fc63"
+            "debugSymbols": {
+                "windows-x64": "f34ee5067be19ed370268b15c53684b7b8aaa867dc800b68931df905d679e31f"
             }
         }
     },
+    "tools": {
+        "packages": {
+            "versions": "1.2.10",
+            "baseUrl": "http://s.sudre.free.fr/Software/files",
+            "label": "Packages.app",
+            "hash": "6afdd25386295974dad8f078b8f1e41cabebd08e72d970bf92f707c7e48b16c9"
+        }
+    },
     "platformConfig": {
-        "macos-x86_64": {
-            "qtVersion": 6,
-            "deploymentTarget": "10.15"
-        },
-        "macos-arm64": {
+        "macos": {
             "qtVersion": 6,
-            "deploymentTarget": "11.0"
-        },
-        "macos-universal": {
-            "qtVersion": 6,
-            "deploymentTarget": "10.15"
+            "deploymentTarget": "11.0",
+            "bundleId": "tld.your-website.obs-plugintemplate"
         },
         "windows-x64": {
             "qtVersion": 6,
             "visualStudio": "Visual Studio 17 2022",
             "platformSDK": "10.0.20348.0"
         },
-        "windows-x86": {
-            "qtVersion": 6,
-            "visualStudio": "Visual Studio 17 2022",
-            "platformSDK": "10.0.20348.0"
-        },
         "linux-x86_64": {
             "qtVersion": 6
-        },
-        "linux-aarch64": {
-            "qtVersion": 6
         }
     },
     "name": "obs-plugintemplate",
     "version": "1.0.0",
     "author": "Your Name Here",
-    "email": "me@mymailhost.com"
+    "website": "https://www.your-website.tld",
+    "email": "me@mymailhost.com",
```

Comment:
We should probably use `example.com` here.

Reply:
I specifically used this because of the recent `example.com` possibly being a malicious site discussion.

Reply:
There must have been some confusion here. mywebsite.com was the possibly malicious one, example.com is IANA reserved for the purpose of being an example domain (like here).

Reply:
> There must have been some confusion here. mywebsite.com was the possibly malicious one, example.com is IANA reserved for the purpose of being an example domain (like here).

What @gxalpha said. `example.com` is specifically reserved and held by the IANA for the purpose of using as an example domain.